docs/en/ingest-management/release-notes/release-notes-7.14.asciidoc

```diff
@@ -86,6 +86,38 @@ malicious users.
 
 ====
 
+[[known-issue-938]]
+.Failure in default policy setup may result in policy with no input 
+[%collapsible]
+====
+
+*Details*
+
+During the setup of default policies, an error may occur, such as a network
+error or connectivity issue, that results in a default policy with no input.
+
+*Impact* +
+
+To resolve this problem, you must:
+
+. Delete the default {fleet-server} policy saved object.
+
+. Go to the {fleet} page in {kib} to force reloading of the object.
+
+//QUESTION: Was there a typo in issue 938, or is it really the default
+//Fleet Server policy saved object that needs to be deleted? Or did you mean
+//to say the default agent policy saved objectd?
+//
+//Can you provide steps that describe how the user deletes the
+//saved object? Can users use the Saved Objects UI? Or do they have to use the
+//API? Please provide exact steps, so I can update this PR with the steps.
+//I could not find a saved object in the UI that matches the policy.
+
+//If we want to add troubleshooting info, I also need to know what the symptom of
+//this problem. Does the agent run and just not ingest data? Are there errors?
```

Comment:
Fleet server will not be able to start using the default fleet server policy, you will see 
```Error - assigned policy does not have fleet-server input``` in fleet server logs